testpilot/frontend/static-src/styles/modules/_experiment-list.scss

```diff
@@ -185,3 +178,13 @@
   height: $grid-unit * 4.8;
   opacity: .9;
 }
+
+[data-hook='experiment-page'] {
```

Comment:
There don't seem to be any norms around per-usage overrides of default styles, so I wasn't really sure where to put this, or what the most appropriate selector would be. Does this warrant a new file?